app/code/Magento/Ui/view/base/web/js/grid/masonry.js

```diff
@@ -0,0 +1,261 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+define([
+    './listing',
+    'jquery',
+    'ko'
+], function (Listing, $, ko) {
+    'use strict';
+
+    return Listing.extend({
+        defaults: {
+            template: 'Magento_Ui/grid/masonry',
+            imports: {
+                rows: '${ $.provider }:data.items',
+                errorMessage: '${ $.provider }:data.errorMessage'
+            },
+            listens: {
+                'rows': 'initComponent'
```

Comment:
Single quotes are redundant for object keys

Reply:
Fixed

Reply:
> Fixed

nope :)

Reply:
now fixed